packages/vim.rb

```diff
@@ -3,29 +3,20 @@
 class Vim < Package
   description 'Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient.'
   homepage 'http://www.vim.org/'
-  version '8.1.1915'
-  source_url 'https://github.com/vim/vim/archive/v8.1.1915.tar.gz'
-  source_sha256 '508bcffd340497d6279fb2d5aa98ff43190cf7983a87fe4838fb780446f900a9'
+  version '8.1.2225'
```

Comment:
Please update `gvim` and `vim_runtime` too.

Reply:
Done in the commit 60ba049. I kept the same philosophy about python 2.7. I will apply the same decision once we will agree on what to do (keep or remove the python 2.7 link)